packages/styled-components/src/index.ts

```diff
@@ -14,6 +14,7 @@ export * from './breakpoints'
 export * from './preflight'
 export * from '@xstyled/system'
 export * from './create'
+export { createX } from './createX'
```

Comment:
It is not OK for me to expose `createX`, I prefer that you change the doc, the only available method should be `createCss()` and using only `x` from it is fine.

Reply:
I've removed this change from the PR.
Let's update the documentation in another PR.